spec.md

```diff
@@ -749,6 +752,7 @@ For success responses, the following fields are defined:
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
 | description | string | A user-facing message displayed to the Platform API client. Can be used to tell the user details about the status of the operation. If present, MUST be a non-empty string. |
+| resource | object | An opaque object of resource message for fetching the result of the operation. If present, MUST be included when `state` is `succeeded`. |
```

Comment:
What do we mean by "If present, MUST be included..." If present where?

Reply:
Also, does the resource object look like anything? or is it just any object? How does the platform know what to do with it?